setup.py

```diff
@@ -48,8 +48,9 @@
     url="https://github.com/ethereum/eth-keyfile",
     include_package_data=True,
     install_requires=[
-        "eth-utils>=2",
         "eth-keys>=0.4.0",
+        "eth-utils>=2",
+        "py_ecc>=5.2.0",
```

Comment:
This has to be `>=5.2.0`, right? We can't open it up more than that?

Reply:
Didn't check earlier, but `5.2.0` is from 2021 with max listed of py37, so figured it was old enough